tests/broker-test.py

```diff
@@ -39,7 +39,8 @@
     _AZURE_CLI,
     authority="https://login.microsoftonline.com/organizations",
     enable_broker_on_mac=True,
-    enable_broker_on_windows=True)
```

Comment:
Thanks, you find this test file, please make sure it is able to invoke broker and you need to manually finish the signin process in the broker prompt.